### DISCUSSION TITLE
Community Meeting #2

### DISCUSSION THREAD

**Discussion by Rumperuu** [markcheret/footnotes#179] [Q&A]
Date: we said May 31 originally, but I can't make that any more so if it's okay with everyone can we do the 29th or June 1st instead?

**Comment by markcheret** [comment]
@Rumperuu Let's check in on 29th separately. The community meeting is going to happen as scheduled, since it's already public and I don't know who has it already booked in their calendars.

**Reply by lolzim** [reply]
Hi, did it happened ? I saw nothing here and, as I had my oral for my CS degree today, I did not have a lot of time in order to check. Anyway, this is now history and I have a little time more than the past 10 days :)
